packages/reports/src/Constants/routes-config.tsx

```diff
@@ -6,12 +6,22 @@ import { localize } from '@deriv/translations';
 const Page404 = React.lazy(() => import(/* webpackChunkName: "404" */ 'Modules/Page404'));
 
 const lazyLoadReportComponent = makeLazyLoader(
-    () => moduleLoader(() => import(/* webpackChunkName: "reports-routes" */ 'Containers')),
+    () => moduleLoader(() => import(/* webpackChunkName: "reports-routes" */ '../Containers')),
     () => <Loading />
 );
 
+type TRoute = {
+    path?: string;
+    component: any;
```

Comment:
Maybe would be better to use this: 
component: React.ComponentType;
In this case this is not any type, it will be Component of any type.

Reply:
Sure, I have followed TRoute implementation from appstore package which has `any` as component type, but it can be changed. I think the real problem is that TRoute in every package has different type for props such as `component`, so I didn't use `React.ComponentType` since it's not used in any of other implementations